service/middleware.go

```diff
@@ -83,3 +96,16 @@ var MiddlewareWithCSRF = func(inner http.Handler) http.Handler {
 
 	return CSRF(Middleware(inner))
 }
+
+func recoverWithSentry(hub *sentry.Hub, w http.ResponseWriter, r *http.Request) {
+	if err := recover(); err != nil {
+		log.Errorf("recover from error: %v", err)
+		w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+		w.WriteHeader(500)
```

Comment:
Looks like no response body is actually sent out here. We might want to add an `InternalServerError` entry to `response.errors` and write it out here?